.github/workflows/post-apiview.yml

```diff
@@ -0,0 +1,27 @@
+name: After APIView
+
+on:
+  check_suite:
```

Comment:
I suspect we will want to do similar to what @raych1 did and make this check_run completed trigger.

Reply:
Won't that result in this triggering more times than is necessary?

Reply:
It depends what you want.  If you'd like to react to the check run completing as quickly as possible, use trigger `check-run` and heavily filter based on the payload, to only allocate an agent when needed.

https://github.com/Azure/azure-rest-api-specs/pull/33158/files#r1994294591

Reply:
@chidozieononiwu could you change to use 'check_run' trigger to align with `spec-gen-sdk` pipelines since API view is a step with them?

Reply:
Do we want this though? APIView wil also run in Swagger and TypeSpec jobs. Seems its best to have this workflow run once at the end of check suite.

Reply:
What is the harm in running it more often as checks finish? It might take some of those longer to run but it would be nice to see some status on the completed ones as they complete.

Reply:
I agree with Wes.  Better to trigger on check_run and filter based on the run name.

Reply:
Am fine with using check_run. One issue though that comes to mind is that we are calling a rate-limited API as part of this workflow. So maybe we should be aiming to run this as little times as possible.

Reply:
Let's keep an eye on the rate limiting and try and ensure we are logging that info so we can see if we are getting close to hitting it or not.

Reply:
It's not implemented yet, but I agree we should add rate limit logging after every GH API call.